ui/src/containers/Dashboards/index.js

```diff
@@ -22,11 +22,12 @@ const renderSpinner = () => (
 );
 
 const renderDashboard = params => (model, index) => (
-  <Tab key={index} label={model.get('title', `Dashboard ${index + 1}`)}>
+  <Tab key={index} label={model.get('title', `Dashboard ${index + 1}`) || ''}>
```

Comment:
The third argument is the default value.
```suggestion
  <Tab key={index} label={model.get('title', `Dashboard ${index + 1}`, '')}>
```